packages/lab/src/metric/MetricContent.tsx

```diff
@@ -89,36 +91,28 @@ export const MetricContent = forwardRef<HTMLDivElement, MetricContentProps>(
     return (
       <div
         {...restProps}
-        className={cx(
-          withBaseName(),
-          {
-            [withBaseName(`direction${capitalise(direction)}`)]: direction,
-            [withBaseName(orientation as string)]: orientation,
-            [withBaseName(`align${capitalise(align)}`)]: align,
-          },
-          className
-        )}
-        ref={ref}
+        className={cx(withBaseName(), className)}
+        ref={handleRef}
+        aria-labelledby={`${titleId || ""} ${subtitleId || ""}`}
       >
-        <div>
+        <div className={withBaseName("value-container")}>
           {indicatorPosition === "start" && icon}
-          <span
-            className={withBaseName("value")}
+          <ValueComponent
             data-testid="metric-value"
             id={valueId}
-            {...headingAriaProps}
+            className={withBaseName("value")}
           >
             {value}
-          </span>
+          </ValueComponent>
           {indicatorPosition === "end" && icon}
         </div>
         {subvalue && (
-          <div
+          <Div
```

Comment:
Text itself defaults to <div>. Would you prefer the use of Text instead of Div here?

Reply:
The reason why we've created Div, Span, P etc was to make it easier for our users to consume the text components. The long version would be `<Text elementType="span" .../>`. Not sure it would make sense to change it really. But it's interesting that you feel this way, maybe the extended text components would not be so useful after-all.